blog/_posts/2025-07-29-temporal-on-aks.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "Streamlining Temporal Worker Deployments on Azure Kubernetes Service (AKS)"
+description: ""
+date: 2025-07-29
+authors:
+  - Steve Womack
+  - Brian Redmond
+categories: developer
+---
+
+## Introduction
+
+[Temporal](https://temporal.io/) is an open source platform that helps developers build and scale resilient Enterprise and AI applications. Complex and long-running processes are easily orchestrated with durable execution, ensuring they never fail or lose state. Every step is tracked in an Event History that lets developers easily observe and debug applications. In this guide, we will help you understand how to run and scale your workers on Azure Kubernetes Service (AKS).
+
+Running Temporal Workers efficiently is crucial for scalable and resilient distributed services. Azure Kubernetes Service (AKS) stands out as a leading platform for hosting Temporal workers, offering tight integration with Azure's ecosystem and built-in auto-scaling capabilities with fault tolerance—critical features for enterprise Temporal deployments.
+
+This guide outlines the process of deploying and operating Temporal Workers on AKS. The instructions are broadly applicable to both self-hosted Temporal installations and Temporal Cloud environments.
+
+Everything you need can be found at the following repo: 
+https://github.com/temporal-community/temporal-on-aks-starter
```

Comment:
Bare URLs won't render on as links on the site, so it needs to be properly formatted.

```suggestion
[https://github.com/temporal-community/temporal-on-aks-starter](https://github.com/temporal-community/temporal-on-aks-starter)
```